OPAL/apk/Dockerfile

```diff
@@ -0,0 +1,76 @@
+# based on RetDec's Dockerfile (https://github.com/avast/retdec)
```

Comment:
Okay for now, but we might want to find a solution that doesn't depend on docker